.tributors

```diff
@@ -2,19 +2,15 @@
     "yarikoptic": {
         "name": "Yaroslav Halchenko",
         "blog": "www.onerussian.com",
-        "email": "debian@onerussian.com",
-        "bio": "Cheers!",
```

Comment:
wondering -- no "bio" and email -- desired or undesired sideeffect of changes?

Reply:
It's because I deleted the file (accidentally added entries from the datalad .zenodo.json) and ran it only to update the contributors file, which doesn't do interaction with GitHub. The metadata there would come from an update of the lookup first.